cmd/virt-chroot/main.go

```diff
@@ -106,7 +106,7 @@ func main() {
 
 		},
 		Run: func(cmd *cobra.Command, args []string) {
-			_, _ = fmt.Fprint(cmd.OutOrStderr(), cmd.UsageString())
+			cmd.Printf(cmd.UsageString())
```

Comment:
@fossedihelm The change from `fmt.Printf` to `cmd.Printf` was for a refactoring reasons, or was that necessary for the new functionality introduced by the PR?

Reply:
It is just a refactoring: `cmd.Printf` make a call to fmt.Fprint(cmd.OutOrStderr(), interface...)